tmp/build/patch-charts.sh

```diff
@@ -355,23 +355,14 @@ function patchKialiOpenShift() {
       N
       a\
 \        - name: kiali-cert\
-\          mountPath: "/kiali-cert"\
-\{\{- if and (.Values.dashboard.user) (.Values.dashboard.passphrase) \}\}\
-\        - name: kiali-secret\
-\          mountPath: "/kiali-secret"\
-\{\{- end \}\}
+\          mountPath: "/kiali-cert"
     }
     /configMap:/ {
       N
       a\
 \      - name: kiali-cert\
 \        secret:\
-\          secretName: kiali-cert-secret\
```

Comment:
We should not be doing this as `oauth` method is configured if either of these are missing.  If they're both specified, we use `login` configuration.  Changing this will alter how auth is configured.